app/src/main/java/com/simplecity/amp_library/ui/presenters/PlayerPresenter.java

```diff
@@ -154,8 +163,13 @@ private void updateFavorite(boolean isFavorite) {
     public void updateTrackInfo() {
         PlayerView view = getView();
         if (view != null) {
+            totalTime = MusicUtils.getDuration();
             view.trackInfoChanged(MusicUtils.getSong());
             view.currentTimeChanged(MusicUtils.getPosition() / 1000);
+            if(SettingsManager.getInstance().preferRemainingTime())             //can be refactored into another function
```

Comment:
Don't leave comments like this in the codebase.. Either refactor into another function, or remove the comment.